spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -978,6 +1044,11 @@ private void validateTables(String source, String dest)
 
   private long expectedFilesCount(String source)
       throws NoSuchDatabaseException, NoSuchTableException, ParseException {
+    return expectedFiles(source).count();
+  }
+
+  private Stream<File> expectedFiles(String source)
```

Comment:
Why do we need this change?

Reply:
Seems like we never really use this as a stream? Could we just make it a list?

Reply:
> Why do we need this change?

Existing cases that get the length of the result are sufficient, but newly added cases need to get the elements.



> Seems like we never really use this as a stream? Could we just make it a list?

Done